File: server/manager/agent-js-sdk/model/LoginTokenModel.js
'use strict';



/**
 * @author  zuoge85 on 15/6/17.
*/
class LoginTokenModel {

    token:String;

    name:String;
    constructor() {

    }

    formObject({token,name}):LoginTokenModel{
        this.token = token;
        this.name = name;
        return this;
    }

    static of({token,name}):LoginTokenModel{
        return new LoginTokenModel().formObject({token,name});
    }

    static form(token:String,name:String):LoginTokenModel{
        return new LoginTokenModel().formObject({token,name});
    }
}

export default LoginTokenModel;

File: server/project/mj-client/src/main/java/majiang/client/portal/admin/model/LoginTokenModel.java
package majiang.client.portal.admin.model;


import org.forkjoin.apikit.core.Message;

/**
 * @author zuoge85 on 15/6/17.
 */
@Message
public class LoginTokenModel {
    private String token;
    private String name;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }

    @Override
    public String toString() {
        return "LoginTokenModel{" +
                "token='" + token + '\'' +
                ", name='" + name + '\'' +
                '}';
    }
}
